core/Cargo.toml

```diff
@@ -14,6 +14,9 @@ thiserror = "1"
 itertools = "0.12"
 lazy_format = "2"
 joinery = "2"
+topological-sort = { version = "0.2.2"}
+once_cell = { version = "1"}
```

Comment:
Use the built-in oncelock https://doc.rust-lang.org/std/sync/struct.OnceLock.html

Reply:
This has been done, and the import removed 😄